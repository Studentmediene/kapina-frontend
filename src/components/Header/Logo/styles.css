.logo {
  width: 12em;
}
