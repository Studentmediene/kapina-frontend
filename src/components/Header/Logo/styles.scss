@import 'variables.scss';

.logo {
  width: $header-height - 15px;
}
