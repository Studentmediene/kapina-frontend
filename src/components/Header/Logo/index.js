import React from 'react';
import Link from 'next/link';

import styles from './styles.scss';

const Logo = () => (
  <Link href="/">
    <img
      src="/assets/RR_logo.png"
      alt="Radio Revolt Logo"
      className={styles.logo}
    />
  </Link>
);

export default Logo;
