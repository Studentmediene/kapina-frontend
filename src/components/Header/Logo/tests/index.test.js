import React from 'react';
import { shallow } from 'enzyme';

import Logo from '../';

describe('<Logo />', () => {
  it('renders correctly', () => {
    const tree = shallow(<Logo />);
    expect(tree).toMatchSnapshot();
  });
});
