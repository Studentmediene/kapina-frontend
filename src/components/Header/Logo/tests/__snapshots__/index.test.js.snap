// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> renders correctly 1`] = `
<Link
  href="/"
>
  <img
    alt="Radio Revolt Logo"
    className="logo"
    src="/assets/RR_logo.png"
  />
</Link>
`;
